google-cloud-monitoring/lib/google/monitoring/v3/uptime_pb.rb

```diff
@@ -8,11 +8,11 @@
 require 'google/protobuf/duration_pb'
 Google::Protobuf::DescriptorPool.generated_pool.build do
   add_message "google.monitoring.v3.InternalChecker" do
-    optional :project_id, :string, 1
-    optional :network, :string, 2
-    optional :gcp_zone, :string, 3
-    optional :checker_id, :string, 4
-    optional :display_name, :string, 5
+    optional :name, :string, 1
```

Comment:
Do we know if this change is going to be reverted?

Reply:
@dazuma @TheRoyalTnetennba @quartzmo ^ anyone know the status of this?

Reply:
The old protocol defined fields 1-5, and this updated proto defines fields 1-4, and 6. I suspect the server is looking for the 6th field to determine if the values are for the previous or current InternalChecker. So the API is backwards compatible, but the client is changed. Are we good to release this?

Reply:
There's an internal discussion on this. The service owners believe that this proto is unused by the public API currently, and that it was included in the GA (v3) protos in error. Hold off on this for another day or so while I get confirmation.

Reply:
Alright, sounds like we're okay with the change since the proto is not used.